src/Microsoft.DotNet.Tools.Compiler.Csc/Program.cs

```diff
@@ -149,9 +158,10 @@ private static IEnumerable<string> TranslateCommonOptions(CommonCompilerOptions
                 commonArgs.Add("-delaysign");
             }
 
-            // TODO: What is this? What does it mean to sign without a key?
-            // Is this "OSS" signing?
-            // if (options.StrongName)
+            if (options.PublicSign == true)
+            {
+                commonArgs.Add("-publicsign");
+            }
```

Comment:
Don't think it matters too much but won't this end up adding `-publicsign` twice in some cases?